client/my-sites/stats/post-trends/index.jsx

```diff
@@ -166,3 +173,17 @@ export default React.createClass( {
 		);
 	}
 } );
+
+export default connect( ( state ) => {
+	const siteId = getSelectedSiteId( state );
+	const query = {
+		startDate: i18n.moment().subtract( 1, 'year' ).startOf( 'month' ).format( 'YYYY-MM-DD' ),
+		endDate: i18n.moment().endOf( 'month' ).format( 'YYYY-MM-DD' )
+	};
+
+	return {
+		requesting: isRequestingSiteStatsForQuery( state, siteId, 'statsStreak', query ),
```

Comment:
Doh good catch.  Probably should rip out `streakList` from being passed in as a prop too!